### DISCUSSION TITLE
A network structure in Multi-Cloud Infrastructure

### DISCUSSION THREAD

**Discussion by yunkon-kim** [cloud-barista/cb-tumblebug#1749] [Show and tell]
This discussion will show and tell **a network structure** based on 
- (#1722) the recently updated MCI concept  and 
- (#1710) Q&A to enhance network feature (KR ver).

![image](https://github.com/user-attachments/assets/fa4f7354-0413-4872-bec4-99381ac57e3a)

The main concept is that Cloud-Barista (CB) will manage a multi-cloud network, which is a logical super CIDR block and subnetted for VPC/VNet and their subnets.

Both CB and CSP will manage VPC/VNet/VPC network and their subnets together. (It could be an area of ​​shared responsibility.)

: Note - `connection config` related issues have not been covered yet.

(Note) Currently, issues and PRs related to this discussion are as follows
* #1672
* #1720
* #1538
* #1747 

I'd like to hear your thoughts on the concept, structure, and terminology.

**Comment by seokho-son** [comment]
@yunkon-kim LGTM!!!!!!!!!!!
